src/JSInterop/Microsoft.JSInterop/src/IJSObjectReference.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.JSInterop
+{
+    /// <summary>
+    /// Represents a reference to a JavaScript object.
+    /// </summary>
+    public interface IJSObjectReference : IAsyncDisposable
```

Comment:
Regarding nullability annotations here, whatever we do for `IJSRuntime` we should do the equivalent thing here.

We have an ongoing conversation about it here: https://github.com/dotnet/aspnetcore/pull/25261/files#r481040491